extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -70,24 +70,26 @@
 
 .i-amphtml-lbg-mask,
 .i-amphtml-lbg-gallery {
-  opacity: 1 !important;
-  background-color: rgba(0,0,0, 1) !important;
   position: absolute !important;
-  top:0 !important;
   left: 0 !important;
   right: 0 !important;
   bottom: 0 !important;
 }
 
+.i-amphtml-lbg-mask {
+  background-color: rgba(0,0,0, 1) !important;
+  top: 0 !important;
```

Comment:
nit: I would say keep `top: 0 !important` in the at line 76, and then override it in the `.i-amphtml-lbg-gallery` rule below. Seeing `top` `right` `bottom` `left` together is a common pattern equivalent to `layout="fill"`, and keeping them together makes the CSS easier to grok (IMO).